draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx

```diff
@@ -0,0 +1,241 @@
+import {
+  ButtonProps,
+  DropdownProps,
+  Heading,
+  Icon,
+} from "@kaizen/component-library"
+import { Tag } from "@kaizen/draft-tag"
+import classNames from "classnames"
+import * as React from "react"
+import NavigationTab, { NavigationTabProps } from "./NavigationTabs"
+import Toolbar from "./Toolbar"
+const styles = require("./TitleBlockZen.scss")
+const leftArrow = require("@kaizen/component-library/icons/arrow-backward.icon.svg")
+  .default
+const rightArrow = require("@kaizen/component-library/icons/arrow-forward.icon.svg")
+  .default
+const hamburgerIcon = require("@kaizen/component-library/icons/hamburger.icon.svg")
+  .default
+
+type Props = {
+  children?: React.ReactNode
+  title: string
+  variant?: Variant
+  breadcrumb?: Breadcrumb
+  avatar?: JSX.Element
+  subtitle?: string
+  sectionTitle?: string
+  sectionTitleDescription?: string
+  handleHamburgerClick?: (event: React.MouseEvent) => void
+  primaryActions?: PrimaryActions
+  secondaryActions?: SecondaryActions
+  navigationTabs?: NavigationTabs
+  textDirection?: TextDirection
+  surveyStatus?: SurveyStatus
+}
+
+type Variant = "admin" | "education" // the default is wisteria bg (AKA "reporting")
+
+type PrimaryActions = Array<React.ReactElement<ButtonProps>>
+
+type SecondaryActions = Array<
+  React.ReactElement<ButtonProps> | React.ReactElement<DropdownProps>
+>
+
+type NavigationTabs = Array<React.ReactElement<NavigationTabProps>>
+
+type TextDirection = "ltr" | "rtl"
+
+type SurveyStatus = {
+  text: string
+  status: "draft" | "live"
+}
+
+type Breadcrumb = {
+  path: string
```

Comment:
My only concern is this not playing nicely with React Router (or whatever frontend routing library Perf/Murmur are using). Is path absolute, or relative?

Reply:
Thanks for this – I'll merge and bring this up with PEX since we'll run into this very soon.

Reply:
@ActuallyACat Just following up – I talked to @sampnorris and he reminded me that as a consumer, Perform is using React Router and needs it to be a link with a click event that they can hook into and prevent the default link behaviour. (The click handler is also part of what gets passed to the breadcrumb already so all good.)